frappe/public/js/frappe/ui/link_preview.js

```diff
@@ -99,18 +99,28 @@ frappe.ui.LinkPreview = class {
 	}
 
 	handle_popover_hide() {
+
+		this.default_timeout = setTimeout(()=> {
```

Comment:
this should be inside show. `handle_popover_hide` is called only once during setup.

Reply:
`handle_popover_hide` was being called each time a popover is shown, but it needs to be called only once during setup, so I changed that as well.